mne/defaults.py

```diff
@@ -12,10 +12,10 @@
                exci='k', ias='k', syst='k', seeg='saddlebrown', dipole='k',
                gof='k', bio='k', ecog='k', hbo='darkblue', hbr='b'),
     units=dict(eeg='uV', grad='fT/cm', mag='fT', eog='uV', misc='AU',
-               seeg='uV', dipole='nAm', gof='GOF', emg='uV', ecg='uV',
+               seeg='mV', dipole='nAm', gof='GOF', emg='uV', ecg='uV',
                bio='uV', ecog='uV', hbo='uM', hbr='uM'),
     scalings=dict(mag=1e15, grad=1e13, eeg=1e6, eog=1e6, emg=1e6, ecg=1e6,
-                  misc=1.0, seeg=1e4, dipole=1e9, gof=1.0, bio=1e6, ecog=1e6,
+                  misc=1.0, seeg=1e3, dipole=1e9, gof=1.0, bio=1e6, ecog=1e6,
```

Comment:
There was a mismatch here between what this `scalings` value was and what we said the `units` were. I think I compensated for it properly in the `ylim`, but some sEEG user should probably try `evoked.plot` to confirm.

Reply:
I just tried. It's good as it is.